wpiutil/src/main/java/edu/wpi/first/util/MsvcRuntimeException.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.util;
+
+public class MsvcRuntimeException extends RuntimeException {
+    private static final long serialVersionUID = -9155939328084105142L;
```

Comment:
Our of curiosity, where did this come from? (And how is it used?)

Reply:
Randomly generated, and something that all custom exceptions in Java shoul have.

Reply:
> (And how is it used?)

> something that all custom exceptions in Java should have

If anyone else was wondering- Looks like it should be defined by every serializable class, and it's accessed via reflection by serializers: https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it. The reason all custom exceptions in Java should have it is that [`Throwable`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Throwable.html) implements `Serializable`, so custom exceptions will automatically be serializable and thus should specify the serial version.

This conversation is good to be resolved. (Since I don't have write access and am not the PR author, I can't mark conversations as resolved)